src/main/proto/ga4gh/rna_quantification.proto

```diff
@@ -70,25 +86,30 @@ message ExpressionLevel {
   // Name
   string name = 2;
 
-  // The associated QuantificationGroup. 
-  string quantification_group_id = 3;
+  // Id of the feature
+  string feature_id = 3;
+
+  // The associated ExpressionLevelGroup. 
+  string expression_level_group_id = 4;
 
   // The number of reads mapped to this feature.
-  float raw_read_count = 4;
+  float raw_read_count = 5;
```

Comment:
For future reference, once we release the API, numbers on the same field *cannot* change. The point of the numbers is that as we add/modify/delete individual fields, data can (largely) be mapped across different versions of the API. Thus, once `raw_read_count` is set to 4, it stays at 4. New fields can be added at the end of the message, given new numbers.